pkg/plugins/policies/meshmetric/plugin/v1alpha1/plugin.go

```diff
@@ -0,0 +1,31 @@
+package v1alpha1
+
+import (
+	"github.com/kumahq/kuma/pkg/core"
+	core_plugins "github.com/kumahq/kuma/pkg/core/plugins"
+	core_mesh "github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
+	core_xds "github.com/kumahq/kuma/pkg/core/xds"
+	"github.com/kumahq/kuma/pkg/plugins/policies/core/matchers"
+	api "github.com/kumahq/kuma/pkg/plugins/policies/meshmetric/api/v1alpha1"
+	xds_context "github.com/kumahq/kuma/pkg/xds/context"
+)
+
+var (
+	_   core_plugins.PolicyPlugin = &plugin{}
+	log                           = core.Log.WithName("MeshMetric")
+)
+
+type plugin struct{}
+
+func NewPlugin() core_plugins.Plugin {
+	return &plugin{}
+}
+
+func (p plugin) MatchedPolicies(dataplane *core_mesh.DataplaneResource, resources xds_context.Resources) (core_xds.TypedMatchingPolicies, error) {
+	return matchers.MatchedPolicies(api.MeshMetricType, dataplane, resources)
+}
+
+func (p plugin) Apply(rs *core_xds.ResourceSet, ctx xds_context.Context, proxy *core_xds.Proxy) error {
+	log.Info("apply is not implemented")
```

Comment:
nitpick: maybe change it to debug until we implement this not to spam logs, or just remove altogether

Reply:
but, you won't be applying this nowhere until we implement it, so I don't think we need to change it now

Reply:
i think it's called anyway it just does not match any DP so it's a noop, can stay but it's a bit annoying

Reply:
changed